docs/element-definitions.md

```diff
@@ -1,1419 +1,1623 @@
 ```json
 {
-  "element": "foo",
+  "element": "asset",
   "meta": {
-    "id": {
+    "classes": {
+      "element": "array",
+      "content": [
+        {
+          "element": "string",
+          "content": "messageBody"
+        }
+      ]
+    }
+  },
+  "attributes": {
+    "contentType": {
       "element": "string",
-      "content": "baz"
+      "content": "application/json"
     }
   },
-  "content": "bar"
+  "content": "{\"foo\": \"bar\"}"
 }
 ```
 
-## Core API Elements
+---
 
-### Href ([String Element][])
+## Data Structure Element types
+
+[API Elements](#api-element-types) and [Parse Result Elements](#parse-result-element-types) are all defined via Data Structure Elements.
+The following table summarizes them very broadly.
+
+<table class="markdown">
+  <thead>
+    <tr>
+      <th>Name</th>
+      <th>Example Element</th>
+      <th>Example value</th>
+    </tr>
+  </thead>
+  <thead>
+    <tr>
+      <th colspan=3>Primitive Element types</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td><a href="#null-element">null</a></td>
+      <td>
+        <pre>{
+  "element": "null"
+}</pre>
+      </td>
+      <td><pre>null</pre></td>
+    </tr>
+    <tr>
+      <td><a href="#boolean-element">boolean</a></td>
+      <td>
+        <pre>{
+  "element": "bool"
+}</pre>
+      </td>
+      <td>
+        <pre>true</pre>
+        <pre>false</pre>
+      </td>
+    </tr>
+    <tr>
+      <td><a href="#number-element">number</a></td>
+      <td>
+        <pre>{
+  "element": "number"
+}</pre>
+      </td>
+      <td>
+        <pre>0</pre>
+        <pre>-1.5</pre>
+        <pre>6.53e-3</pre>
+      </td>
+    </tr>
+    <tr>
+      <td><a href="#string-element">string</a></td>
+      <td>
+        <pre>{
+  "element": "string"
+}</pre>
+      </td>
+      <td>
+        <pre>"Hello world"</pre>
+        <pre>""</pre>
+      </td>
+    </tr>
+  </tbody>
+  <thead>
+    <tr>
+      <th colspan=3>Structured Type Elements</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td><a href="#array-element">array</a></td>
+      <td>
+        <pre>{
+  "element": "array"
+}</pre>
+      </td>
+      <td>
+        <pre>[]</pre>
+        <pre>[42, "Hello world!"]</pre>
+      </td>
+    </tr>
+    <tr>
+      <td><a href="#object-element">object</a> & <a href="#member-element">member</a></td>
+      <td>
+        <pre>{
+  "element": "object",
+  "content": [
+    {
+      "element": "member",
+      "content": {
+        "key": {
+          "element": "string",
+          "content": "foo"
+        },
+        "value": {
+          "element": "string"
+        }
+      }
+    },
+  ]
+}</pre>
+      </td>
+      <td>
+        <pre>{}</pre>
+        <pre>{
+  "foo": "Hey!"
+}</pre>
+        <pre>{
+  "bar": true
+}</pre>
```

Comment:
Let's separate out the member element from this. For the example value of the member element, we should say that there is no JSON equivalent without the context but we can show them the part of an object that it represents. `"key": value`